js-core/homeworks/homework-14/src/main.js

```diff
@@ -0,0 +1,209 @@
+/*
+TASK 0. Найдите числа которые повторяются нечетное количество раз
+в массиве
+  solution([12, 23, 34, 12, 12, 23, 12, 45]) -> [34 45]
+  solution([4, 4, 100, 5000, 4, 4, 4, 4, 4, 100, 100,]) -> [4 100 5000]
+  solution([3, 3, 4, 6, 4, 5, 9, 9, 21, 9]) -> [6 5 9 21]
+  solution([4, 8, 15, 16, 23, 42, 4, 15, 42, 42]) -> [8 16 23 42]
+  solution([2, 2, 44, 44]) => []
+*/
+
+
+function solution(arr){
+    let _temp = {}
+    arr.forEach( function(element, index) {
+      if(_temp[element]){
+        _temp[element]++ ;
+      }else{
+        _temp[element] = 1;
+      }      
+      // statements
+    });
+
+    let res = [];
+
+    Object.keys(_temp).forEach(function(value){
```

Comment:
your solution looks similar to Sergey are you worked a pair-programming?

Reply:
Nope....

Reply:
Is solution wrong& what I must to do with your comment?